frontend/.gitignore

```diff
@@ -12,6 +12,7 @@ dist
 dist-ssr
 *.local
 .env
+.DS_Store
```

Comment:
no need to ignore it in a subfolder once it's ignored in the root .gitignore file